packages/manager/apps/telecom/src/app/telecom/pack/migration/meeting/pack-migration-meeting.controller.js

```diff
@@ -0,0 +1,131 @@
+import get from 'lodash/get';
+
+export default class TelecomPackMigrationMeetingCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $translate,
+    OvhApiConnectivityEligibility,
+    OvhApiConnectivityEligibilitySearch,
+    TucPackMigrationProcess,
+    TucToast,
+  ) {
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.OvhApiConnectivityEligibility = OvhApiConnectivityEligibility;
+    this.OvhApiConnectivityEligibilitySearch = OvhApiConnectivityEligibilitySearch;
+    this.TucPackMigrationProcess = TucPackMigrationProcess;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.selectedMeeting = {
+      meetingSlots: {
+        fakeMeeting: false,
+        slot: null,
+      },
+    };
+    this.contactName = null;
+    this.showMeetingSlots = false;
+    this.meetingSlots = {};
+
+    this.process = this.TucPackMigrationProcess.getMigrationProcess();
+    this.buildingReference = this.process.selectedOffer.buildingReference;
+
+    this.loading = true;
+    this.meetings = [];
+    return this.OvhApiConnectivityEligibility.v6()
+      .testBuilding(this.$scope, { building: this.buildingReference })
+      .then((eligibility) => {
+        if (eligibility.result) {
+          this.eligibilityReference = eligibility.result.eligibilityReference;
+          this.productCode = eligibility.result.offers[0].product.code;
```

Comment:
Can you avoid binding productCode to 'this' ? You can use it as a local variable by declaring it as let